sdks/java/extensions/python/build.gradle

```diff
@@ -30,6 +30,7 @@ dependencies {
     implementation project(path: ":model:pipeline", configuration: "shadow")
     implementation project(path: ":runners:core-construction-java")
     implementation project(path: ":sdks:java:core", configuration: "shadow")
+    implementation project(path: ":sdks:java:transform-service:launcher", configuration: "shadow")
```

Comment:
If `PythonExternalTransformOptions.getUseTransformService` is false, do we still need this depenency? i.e.. Should this be "runtimeOnly" or some optional dependencies

Reply:
We need the dependency to compile. Also, ":sdks:java:transform-service:launcher" is intentionally very light weight. So I don't think there's much overhead on adding this dependency.